README.md

```diff
@@ -252,7 +233,7 @@ Yes, the [material-foundation](https://github.com/material-foundation) organizat
 
 ## Contributing
 
-We gladly welcome contributions! If you have found a bug, have questions, or wish to contribute, please follow our [Contributing Guidelines](https://github.com/material-foundation/material-remixer-android/blob/develop/CONTRIBUTING.md).
+We gladly welcome contributions! If you have found a bug, have questions, or wish to contribute, please follow our [Contributing Guidelines](CONTRIBUTING.md) and read the [High-level Project Structure](PROJECT_STRUCTURE.md).
```

Comment:
your README could server as a better-formatted table of contents. If you plan to have many of these files, you may want to just list them in bullet point format.